### DISCUSSION TITLE
Manual Update button

### DISCUSSION THREAD

**Discussion by efloresgil** [tughi/aggregator-android#74] [Ideas]
Hello, thanks for all your work on the app.

I would like the app to allow you to manually refresh the feed, with a button, or a gesture like dragging down, a UI component to do so.

Currently you can set it up so that navigating out and back in could do it but this is a bit inconvenient.

**Comment by RequestPrivacy** [comment]
That's the only feature I really, really miss in the app!

**Comment by tughi** [comment]
Currently the only way to manually update a feed is in the feeds tab.

![image](https://user-images.githubusercontent.com/421092/216755133-78e8c081-4bc5-4bd7-9f6a-0f7d6fdc1baa.png)
